DOCS-3002: Add instructions for physically connecting components to how-tos (#3535)
viamrobotics/docs: docs/how-tos/detect-color.md

{{% /expand%}}
{{% expand "Step 3: Configure your webcam" %}}

> First, make sure to connect your webcam to your machine if it's not already connected (like with an inbuilt webcam on a Macbook).

Reviewer: ```suggestion
First, make sure to connect your webcam to your machine if it's not already connected (like with an inbuilt laptop webcam).
```
